src/web/ballot.js

```diff
@@ -191,10 +199,10 @@ export function lookup_ballot_user_rho(acct, balloturi, votersuri) {
       for (@ballot <- valueCh) {
           lookup!( \`${votersuri}\` , *valueCh) |
           for (@accts <- valueCh) {   
-            return!({"registered": accts.contain("${acct}") ,"ballot": ballot})
+            return!({"registered": accts.contains("${acct}") ,"ballot": ballot})
```

Comment:
interpolating `acct` this way risks injection. Try this instead:

```suggestion
            return!({"registered": accts.contains(${JSON.stringify(acct)}) ,"ballot": ballot})
```